src/iconsManifest/supportedExtensions.ts

```diff
@@ -1319,6 +1319,13 @@ export const extensions: IFileCollection = {
       light: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'denoify',
+      filenamesGlob: ['denoify.config'],
```

Comment:
Gotta add the `.denoifyrc` file into here, otherwise it won't be picked up on by the icons.
```suggestion
      extensions: ['.denoifyrc'],
      filenamesGlob: ['denoify.config'],
```

Reply:
Note: Putting that there instead of doing it myself to ensure @GervinFung still has a high level of contribution to it 👍

Reply:
I just wanted to be consistent with the Denoify documentation: https://docs.denoify.land/build-options  

We allow `denoify.config.js` or `denoify.config.json` but thank you if I got confused and made a mistake!

Reply:
> Note: Putting that there instead of doing it myself to ensure @GervinFung still has a high level of contribution to it +1

Sorry Im late, just woke up 2 mins ago. Some other stuffs that's happening caught my attention, so wasn't doing much. Ok, u can do it urself too, but anyhow, since u said it, Ill do something with it